code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1230,7 +1230,7 @@
 	result_amount = 3
 
 /datum/chemical_reaction/drinks/nuka_cola
-	name = "Nuka Cola"
+	name = "Nuclear Cola"

```

Comment:
Well at least add a silly description about how NanoTrasen had to change the name due to a copyright lawsuit or something. This way we not only keep the easter egg but it's enhanced a bit since we all know how sue-happy Bethesda's lawyers are.